src/main/java/uk/gov/hmcts/reform/camunda/bpm/config/EventHandlerConfiguration.java

```diff
@@ -46,6 +45,7 @@ public void onTaskCreatedEvent(DelegateTask delegateTask) {
             );
 
         }
+        delegateTask.setVariableLocal(CFT_TASK_STATE_LOCAL_VARIABLE_NAME, "unconfigured");
```

Comment:
Maybe put a log message too so we can track it if needed. 
```suggestion
        LOG.info("Setting {} state to unconfigured for Task id: {}", CFT_TASK_STATE_LOCAL_VARIABLE_NAME, delegateTask.getId());
        delegateTask.setVariableLocal(CFT_TASK_STATE_LOCAL_VARIABLE_NAME, "unconfigured");
```

Reply:
Added